### PR TITLE
feat: add default breakpoint for fast-layouts-react column

### PR DESCRIPTION
**Pull request by JamesBurnside**
# Description

Add default breakpoint to the column component fast-layouts-react

## Motivation & context

When performing server side rendering there is no way to set the column width,span&order.
#2398 

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [x] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [x] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-layouts-react/src/column/column.spec.tsx [hunk 1]
```diff
@@ -241,6 +242,53 @@ describe("Column", (): void => {
...
+
+    test("column should use a default breakpoint of zero when the DOM is unavailable and default breakpoint is unset", () => {
+        // make DOM unavailable for test
+        const windowSpy: jest.SpyInstance<any> = jest.spyOn(
+            global as any,
+            "window",
+            "get"
+        );
+        windowSpy.mockImplementation(() => ({ undefined }));
+        expect(canUseDOM()).toEqual(false);
```
Comment (chrisdholt) [inline packages/fast-layouts-react/src/column/column.spec.tsx]: We had some issues with tests failing with this implementation. Can you check out the breakpoint tracker test updates in this commit and see if we can leverage that pattern here? https://github.com/microsoft/fast-dna/commit/25633f6e63c69a565fb65e654607b6a87b7fe9d2

**Comment by chrisdholt** [comment]
@JamesBurnside Just an FYI that I updated your PR message - thanks!

File: packages/fast-layouts-react/src/column/column.spec.tsx (hunk 1 above)
Comment (JamesBurnside) [reply packages/fast-layouts-react/src/column/column.spec.tsx]: That's much cleaner as well, yep I'll update

File: packages/fast-layouts-react/src/column/column.spec.tsx [hunk 2]
```diff
@@ -241,9 +252,50 @@ describe("Column", (): void => {
...
+        const rendered: any = shallow(
+            <Column
+                defaultBreakpoint={2}
+                span={[7, 8, 9]}
+                row={[10, 11, 12]}
+                order={[13, 14, 15]}
+            />
+        );
+
+        // when breakpoint is zero, GridColumn, GridRowStart and order styles should use the default breakpoint value
```
Comment (chrisdholt) [inline packages/fast-layouts-react/src/column/column.spec.tsx]:
```suggestion
        // when the DOM is unavailable, GridColumn, GridRowStart and order styles should use the default breakpoint value
```

**Pull request merged**